attractions/rollup.config.js

```diff
@@ -77,6 +79,17 @@ export default [
         preprocess: [prependTagOption(icons), sveltePreprocess],
       }),
       resolve(),
+      sveld({
+        typesOptions: {
+          preamble: 'export * as utils from "./utils";\n',
+        },
+      }),
+      {
+        name: 'utilsTypings',
+        writeBundle() {
+          spawnSync('tsc', { shell: true });
```

Comment:
Isn't there a nice TypeScript plugin for Rollup?

Reply:
Yes, but it will affect the bundle. It cannot simply do a side effect and leave the rest of the build as it is. In fact, they explicitly state that it will ignore the `emitDeclarationOnly` (among other options) and set it to `false`